policyengine_us_data/datasets/cps/cps.py

```diff
@@ -2013,11 +2013,14 @@ class Pooled_3_Year_CPS_2023(PooledCPS):
 
 
 if __name__ == "__main__":
-    CPS_2021().generate()
-    CPS_2022().generate()
-    CPS_2023().generate()
-    CPS_2024().generate()
-    CPS_2021_Full().generate()
-    CPS_2022_Full().generate()
-    CPS_2023_Full().generate()
-    Pooled_3_Year_CPS_2023().generate()
+    if test_lite:
+        CPS_2024().generate()
```

Comment:
`test_lite` is a great idea, but don't you need to generate CPS_2023 first? I don't see the generate method called when it's pulled in for CPS_2024:

```
        if self.raw_cps is None:
            # Extrapolate from CPS 2023

            cps_2023 = CPS_2023(require=True)
            arrays = cps_2023.load_dataset()
            arrays = uprate_cps_data(arrays, 2023, self.time_period)
            self.save_dataset(arrays)
            return
```

Reply:
It's called in the require=True bit. Again honestly we should refactor this repo to be just functions for sure.